tests/ovs_test.go

```diff
@@ -35,4 +38,52 @@ var _ = Describe("ovs-cni", func() {
 			})
 		})
 	})
+
+	Describe("pod ovs-bridge connectivity tests", func() {
+		const bridgeName = "br-test"
+		Context("When creating ovs-bridge on the host node", func() {
```

Comment:
We should use past tense. The test inside is not running while we are creating the bridge, it is running "when an OVS bridge is configured on a node"

Reply:
ack. 
Done